.env.example

```diff
@@ -1 +1,2 @@
-TOKEN=
\ No newline at end of file
+TOKEN=
+WELCOME_CHANNEL=
```

Comment:
Make this a constant

Reply:
No, that will make it hard to run the bot locally